### ISSUE TITLE
Update the Eclipse MicroProfile Features image on homepage

### ISSUE THREAD

**Issue by mbroz2** [OpenLiberty/openliberty.io#1405]
Currently, the Eclipse MicroProfile Features image on homepage is a single image, and it's already out of date:
![image.png](https://images.zenhubusercontent.com/59e8c568a468affe6df751e7/2e1907fa-5ca8-4c26-ad62-f91a0e83857a)

Instead, this 'image' should just be a bunch of html elements styled with css, that way we can update  them (aka, add new ones) as necessary without needing to wait for a new svg/png to be created.

As part of this work, we'll want to make sure that all the features of the latest MP are shown, and this will likely also necessitate a structural/layout change to the boxes.

@gcharters may have some insight on how to best layout the boxes, similar to as seen here from his deck: 
![image.png](https://images.zenhubusercontent.com/59e8c568a468affe6df751e7/a5b89987-ef7e-4f3e-bb73-6b19f621b2d9)

@lauracowen we should also have a consistent way to represent the features, which will likely mean updating our existing images around MP, like here: https://www.openliberty.io/docs/ref/general/#microprofile.html

**Comment by lauracowen** [comment]
Yes, I agree - we need an easy-to-update format that we can use in both places on the site. Ability to click individual bits of it would be nice.

I put something in the homepage design feedback about this (but no idea now if that was in the in the issue/Slack/etc... :( ) because the existing image was already out-of-date before we published it on the homepage. I think I said something like Graham's diagram but that it's not obvious what the side bar means - that's a specification detail (ie the side bar stuff isn't required in a server implementation to call it MicroProfile 3.0), not a usage thing (just want to understand what the features are for). I don't think we should include GraphQL or LRA at the moment as we don't have GA for either. The Reactive ones and Context Propagation should be added to the bottom row, I think.

**Comment by mbroz2** [comment]
I agree that we should not include anything that we don't have GA features for (unless we want to do a 'coming soon' section to help show our continued dedication to the spec).

I believe I've seen @gcharters also have row headers for these charts in the past, but I don't know what the current best-practice format is for displaying/grouping them is.  I'll try and get input from others on how to best show the features, but in the interim, feel free to post ideas here as well.

**Comment by lauracowen** [comment]
Yeah, row headers...the problem is a lack of agreement on what the headers should be. I like @gcharters' but he uses them with explanation in his talks and they can be misinterpreted without the explanation... So we just removed them from the diagram for now.

**Comment by mbroz2** [comment]
I agree that for the homepage, having a simpler diagram with no headers will likely be the way to go so as to not over-complicate the content and keep the homepage clean.  However, for the MP docs page, it may be worthwhile to have a more 'complete' picture of the MP features, including the headers since the documentation around the graphic can clarify the headers.

**Comment by mbroz2** [comment]
Current decision is to not include the mpReactiveMessaging-1.0 mpReactiveStreams-1.0 mpContextPropagation-1.0 features in the homepage graphic.  As such, this issue, for now, will only focus on transforming the image into an html/css representation.

**Comment by lauracowen** [comment]
Though we will need to include those in the diagram on the internal page, so I think it's worth adding them and just having two flavours of the diagram on the site.
